code/game/objects/items/food/egg.dm

```diff
@@ -107,7 +107,7 @@ GLOBAL_VAR_INIT(chicks_from_eggs, 0)
 	if(. == SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN)
 		return
 
-	if(!istype(target, /obj/machinery/griddle))
+	if(!istype(target, /obj/machinery/griddle, /obj/structure/table))
```

Comment:
Few things:
- This is improper istype syntax
- This isn't how we do non-modular edits here, please see [the modularity README](https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/modular_skyrat/readme.md)
- I would suggest taking this upstream first
- You seem to have committed this to your master branch, which is bad